vue-vuetify/src/util/composition.ts

```diff
@@ -275,3 +296,13 @@ export const useNested = (element: false | 'array' | 'object'): NestedInfo => {
   }
   return nestedInfo;
 };
+
+
+export const useBlurHandler = (control: any) => {
```

Comment:
We should type the `control` here to at least declare the properties, see for example `useVuetifyLabel`

Reply:
Now that the renderers can access `handleBlur` directly from the `useVuetifyControl` there is no need to pass the control into the blur handler.